### DISCUSSION TITLE
buddhism/2023/eat/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#72] [General]
# buddhism/2023/eat/

https://shanenull.com/buddhism/2023/eat/

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2021/food/

**Comment by shane0** [comment]
This offering is the fruit of many's work and care; Reflecting on my conduct, "Am I worthy of a share?" Be wary of the poisons, the foremost being greed; Seeing food as medicine, I take only what I need, Solely for cultivation, to realize the Buddha Way. With gratitude I accept this offering today.

**Comment by shane0** [comment]
# The Five Contemplations

1. This food is a gift of the Earth, the sky, numerous living beings, and much hard and loving work.
2. May we eat with mindfulness and gratitude so as to be worthy to receive this food.
3. May we recognize and transform unwhole- some mental formations, especially our greed, and learn to eat with moderation.
4. May we keep our compassion alive by eat- ing in such a way that reduces the suffering of living beings, stops contributing to climate change, and heals and preserves our pre- cious planet.
5. We accept this food so that we may nurture our brotherhood and sisterhood, build our community, and nourish our ideal of serving all living beings.

**Comment by shane0** [comment]
# workflow

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2024/workflow/
